cmd/apiserver/app/server/run_server.go

```diff
@@ -42,7 +42,8 @@ func RunServer(opts *ServiceCatalogServerOptions) error {
 func runTPRServer(opts *ServiceCatalogServerOptions) error {
 	tprOpts := opts.TPROptions
 	glog.Infoln("Installing TPR types to the cluster")
-	if err := tpr.InstallTypes(tprOpts.clIface); err != nil {
+	tprInstaller := tpr.NewInstallerTPRs(tprOpts.clIface)
```

Comment:
I turned the `Installer` into an object largely for testing sanity, despite it only has a single method at the moment. Running the server requires a number of dependencies, and `Installer` is one of them, it being an object will allows us to create double and handle dependency injection much easier, thus, make our life easier as the project grow larger.
I made it into an object